src/swift/Sources/Models/TypeModel.swift

```diff
@@ -53,6 +48,10 @@ struct TypeModel {
         isArray = true
     }
 
+    init(from source: String) {
+        name = source
+    }
```

Comment:
What exactly is the 'source'?

Reply:
This is a fallback if you just wanted to specify a type annotation by name. It's used when we don't have a strongly typed version and results in non-clickable text.